kratos/tests/test_model_part.py

```diff
@@ -296,6 +296,39 @@ def test_model_part_properties(self):
         self.assertEqual(inlets_model_part.NumberOfProperties(), 5)
         self.assertEqual(model_part.NumberOfProperties(), 7)
 
+        # Testing subproperties
+        prop_4 = model_part.GetProperties()[4]
+        prop_4.AddSubProperties(model_part.GetProperties()[3])
+        self.assertEqual(prop_4.NumberOfSubproperties(), 1)
+        self.assertEqual(prop_4.HasSubProperties(3), True)
+        self.assertEqual(prop_4.HasSubProperties(120), False)
+        self.assertEqual(prop_4.HasSubProperties("4.3"), True)

```

Comment:
shouldn't the input here be `"3"`?
What happens if you give `"5.3"?

Reply:
"" is by adress, integer is direct get

Reply:
5.3  is false of course

Reply:
> "" is by adress, integer is direct get

yes but why do you have to give the "parent" id too?

Reply:
According to what @pooyan-dadvand said, yes

Reply:
ok then fine from my side, no more complaints if @pooyan-dadvand is ok :)

Reply:
In fact, I didn't like it too much either. IMO we don't need to pass the parent id too.

Reply:
Then I remove the parent?

![imagen](https://user-images.githubusercontent.com/19991680/64597273-b8886a80-d3b5-11e9-9f83-c7e32997b774.png)


I did it as you ask me

Reply:
I understand that from the modelpart the name is the full one, and from the properties the short one.

Reply:
But this was from Json and Modelpart. So you need to know the parent properties.

Reply:
OK, so it is done, right now it is like that